### PR TITLE
add new term: 'Decreased middle cerebral artery pulsatility index'

### LINKED ISSUE #7367: Decreased middle cerebral artery pulsatility index
**Preferred term label:**
Decreased middle cerebral artery pulsatility index

**Synonyms**

**Definition (free text, please give PubMed ID)**
Abnormally reduced blood flow pulsatility in in the middle cerebral artery (MCA). The pulsatility index (PI) is calculated using the following equation PI  =  (peak systolic velocity−end-diastolic velocity)/mean velocity. 

Comment
Arterial blood flow pulsatility, as assessed by Doppler, is influenced by many factors, including the differences in resistance between vascular beds upstream and downstream of the artery assessed, intrinsic properties of the artery, fetal heart rate and cardiac output. In the presence of structural heart disease, reduced cerebral oxygen content, abnormal cardiac output, anatomical substrate that impedes cerebral blood flow and differences in systemic and placental vascular resistance may all potentially contribute to changes in middle cerebral artery pulsatility index (MCA-PI). Fetuses with severe left heart obstruction, including hypoplastic left heart syndrome and isolated coarctation associated with retrograde distal arch flow, have altered cerebral blood flow with reduced MCA-PI. These changes in cerebral blood flow potentially contribute to the neurodevelopmental abnormalities frequently observed in affected infants and children.

PMID: 23456797

**Parent term (use hpo.jax.org/app)**
Abnormal fetal cardiovascular physiology

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
close #7367

### CODE REVIEW

**Comment by pnrobinson** [comment]
Looks good!
